.github/workflows/e2e-k3d.yml

```diff
@@ -47,6 +47,13 @@ jobs:
           curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
           k3d version
       - uses: azure/setup-helm@v4
+      - name: Validate submitToEnaProduction is not true in values.yaml
```

Comment:
Do we need this check? As far as I can imagine we should never even check secrets that would let us do this in?

Reply:
So I have secrets checked in for the ena accounts and I do like to be able to test submission to ena dev on the previews (you use the same credentials for prod and dev sadly). Previously I hard coded which ena instance we talked to make sure it could never happen that we talked to ena prod on a preview but now I need to add a flag for pathoplexus.